lib/Mojo/Transaction/WebSocket.pm

```diff
@@ -86,6 +86,8 @@ sub parse_message {
   # Append chunk and check message size
   $self->{op} = $op unless exists $self->{op};
```

Comment:
This would remove the need for the `delete` below.
```perl
@{$self}{qw(op pmc)} = ($op, $self->compressed && $frame->[1])
  unless exists $self->{op};
```